pkg/kubelet/images/image_manager.go

```diff
@@ -173,13 +174,14 @@ func (m *imageManager) EnsureImageExists(ctx context.Context, pod *v1.Pod, conta
 // applyDefaultImageTag parses a docker image string, if it doesn't contain any tag or digest,
 // a default tag will be applied.
 func applyDefaultImageTag(image string) (string, error) {
-	named, err := dockerref.ParseNormalizedNamed(image)
+	_, tag, _, err := parsers.ParseImageName(image)
```

Comment:
this should continue paying attention to the returned digest
```suggestion
	_, tag, digest, err := parsers.ParseImageName(image)
```

Reply:
updated.